src/porowiec-michal/Fsd.porowiec.michal.Cs/Fsd.porowiec.michal.Cs1/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.porowiec.michal.Cs1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Main start = new Main();
```

Comment:
Main to nazwa która nie sugeruje co się dzieje w środku. Nazwij inaczej np. StringAnalyzer itp aby inny programista wiedział o co chodzi